docs/scripts_md/imaging_upload_file.md

```diff
@@ -0,0 +1,93 @@
+# NAME
+
+imaging\_upload\_file.pl -- a single step script for the imaging pre-processing
+and insertion pipeline sequence
+
+# SYNOPSIS
+
+perl imaging\_upload\_file.pl &lt;/path/to/UploadedFile> \`\[options\]\`
+
+Available options are:
+
+\-profile      : name of the config file in
+                `../dicom-archive/.loris_mri`
+
+\-upload\_id    : The upload ID of the given scan uploaded
+
+\-verbose      : if set, be verbose
+
+# DESCRIPTION
+
+The program does the following
+
+\- Gets the location of the uploaded file (.zip,.tar.gz or .tgz)
+
+\- Unzips the uploaded file
+
+\- Sources the Environment
+
+\- Uses the ImagingUpload class to :
+   1) Validate the uploaded file   (set the validation to true)
+   2) Run dicomtar.pl on the file  (set the dicomtar to true)
+   3) Run tarchiveLoader on the file (set the minc-created to true)
+   4) Removes the uploaded file once the previous steps have completed
+   5) Update the mri\_upload table
+
+## Methods
+
+### getPnameUsingUploadID()
+
+Function that gets the patient name using the upload ID
+
+INPUT: $upload\_id: The upload ID
+
+Returns: $patient\_name : The patient name
+
+### getFilePathUsingUploadID()
+
+Functions that gets the file path from the \`mri\_upload\` table using the upload
+ID
+
+INPUT:  $upload\_id: The upload ID
+
+RETURNS: $file\_path : The full path to the uploaded file
+
+### getNumberOfMincFiles()
+
+Function that gets the count of minc files created and inserted using the
+upload ID
+
+INPUT:  $upload\_id: The upload ID
+
+RETURNS: $minc\_created and $minc\_inserted: count of minc files created and
+inserted
+
+### spool()
+
+Function that calls the Notify->spool function to log all messages
+
+INPUTS:
+ - $this      : Reference to the class
+ - $message   : Message to be logged in the database
+ - $error     : if 'Y' it's an error log , 'N' otherwise
```

Comment:
Should be consistent on either using upper case or lower case to start the description. In this section, the first letter of all descriptions are upper case except for the $error option (if).